src/clients/c++/CMakeLists.txt

```diff
@@ -109,6 +109,23 @@ install(
   RUNTIME DESTINATION bin
 )
 
+#
+# image_shm_client
+#
+find_package(OpenCV REQUIRED)
+add_executable(image_shm_client image_shm_client.cc)
```

Comment:
Instead of copying image_client should just add an option to the existing image_client to use shared memory for input and outputs.

Reply:
Done